README.rst

```diff
@@ -7,105 +7,57 @@ Wikipedia.
-        <diff.chars_added>: 8
-        <user.age>: 71821407
-        <user.is_anon>: False
-        <user.is_bot>: False
+Using a scorer_model to score a revision:
+
+    >>> import mwapi
+    >>> from revscoring import ScorerModel
+    >>> from revscoring.extractors import APIExtractor
+    >>>
+    >>> with open("models/enwiki.damaging.linear_svc.model") as f:
+    ...     scorer_model = ScorerModel.load(f)
+    ...
+    >>> extractor = APIExtractor(mwapi.Session(host="https://en.wikipedia.org",
+    ...                                        user_agent="revscoring demo"))
+    >>>
+    >>> feature_values = extractor.extract(123456789, scorer_model.features)
+    >>>
+    >>> print(scorer_model.score(feature_values))
+    {'prediction': True, 'probability': {False: 0.4694409344514984, True: 0.5305590655485017}}
 
 
 Installation
-================
-
-Packages
----------
-In order to use this, you need to install a few packages first:
-
-You might need to install some other dependencies depending on your operating
-system.  Try using the packages,
-
-``sudo apt-get install python3-dev python3-numpy python3-scipy g++ gfortran liblapack-dev libopenblas-dev myspell-pt myspell-fa myspell-en-au myspell-en-gb myspell-en-us myspell-en-za myspell-fr myspell-es hunspell-vi myspell-he``
-
-If you're on Ubuntu, you might also be able to install an Indonesian dictionary:
-
-``sudo apt-get install aspell-id``
-
-Virtualenv users, please note that you'll have to use the --system-site-packages
-option if you install scipy and numpy via apt-get.  You can also use pip3 within
-your virtualenv.
-
-Python modules
-----------------
-If you need the Python package installer,
-
-``sudo easy_install3 pip``
-
-Then, install this module,
-
-``pip3 install --user revscoring``
-
-You'll need to download NLTK data in order to make use of language features.
-
-``python3 -m nltk.downloader stopwords``
+============
+The easiest way to install `revscoring` is via the Python package installer
+(pip).
+
+``pip install revscoring``
+
+You may find that some of `revscorings` dependencies fail to compile (namely
+`scipy`, `numpy` and `sklearn`).  In that case, you'll need to install some
+dependencies in your operating system.
+
+Ubuntu & Debian:
+  Run ``sudo apt-get install python3-dev g++ gfortran liblapack-dev libopenblas-dev``
+Windows:
+  'TODO'
+MacOS:
+  'TODO'
+
+Finally, in order to make use of language features, you'll need to download
+some NLTK data.  The following command will get the necessary corpus.
+
+``python -m nltk.downloader stopwords``
+
+You'll also need to install [enchant](https://enchant.org) compatible
+dictionaries of the languages you'd like to use.  We recommend the following:
+
+* ``languages.english``:  myspell-en-us myspell-en-gb myspell-en-au
+* ``languages.french``: myspell-fr
+* ``languages.spanish``: myspell-es
+* ``languages.vietnamese``: hunspell-vi
+* ``languages.hebrew``: myspell-he
+* ``languages.portugueses``: myspell-pt
```

Comment:
Unwanted plural?
BTW: is there anything for dealing with both pt-PT and pt-BR? (as in en-US vs en-GB vs en-AU)